config/BUILD

```diff
@@ -0,0 +1,61 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Fabulous!  This looks great!  This looks great to merge, but I'd like to try to get the .bzl file working in a follow up PR.  Essentially the purpose of the .bzl file is so that you don't have to type "py_library" over and over :).  

So for the .bzl we have the function:
```
def BuildLanguageConfig(language, extra_deps_for_normalizer = [], extra_data_for_normalizer = []):
  py_library(
    name = language,
    srcs = [language + ".py"]
)
```

Then in this file, we load that in and instead of having a py_library for each language, we have:
```
BuildLanguageConfig("af")
BuildLanguageConfig("am")
...
```

That should work the same way as what you have here!  Then you shouldn't need to make any changes to the py_binary deps or the visibility!

Reply:
That makes sense. I'll merge this now, but keep this in mind for a follow-up PR!